app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -166,7 +170,16 @@ $(document).ready(function($) {
       _alert('You do not have any transactions to payout.  Please add payees to the form.', 'error');
       return;
     }
+    var usernames = $('.username');
```

Comment:
encourage moving to let / const as opposed to var ? 
cause we've got the ES6 support coming in this week